DevelopGitHub/GitHubExtension/GitHubExtension.Tools.LocalizationTool/View/MainWindow.xaml

```diff
@@ -0,0 +1,104 @@
+                <DataGridTextColumn Header="En" Visibility="{Binding ElementName=En, Path=IsChecked}" Binding="{Binding En, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"/>
+                <DataGridTextColumn Header="Ru" Visibility="{Binding ElementName=Ru, Path=IsChecked}" Binding="{Binding Ru, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"/>
+            </DataGrid.Columns>
+        </DataGrid>
+        <Grid Grid.Column="1" Margin="0" Grid.Row="1" Height="349" VerticalAlignment="Top">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="2*"/>
+                <ColumnDefinition/>
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+                <RowDefinition Height="1*"/>
+            </Grid.RowDefinitions>
+            <Label Content="UK" Margin="6,6,3,6"/>
+            <CheckBox Name="Uk" Content="&#xA;" Grid.Row="0" Grid.Column="1" Margin="31,10,32,10" HorizontalContentAlignment="Center" VerticalContentAlignment="Center" IsChecked="True" HorizontalAlignment="Center" VerticalAlignment="Center"/>
+            <Label Content="US" Margin="6,6,3,6" Grid.Row="1" Grid.Column="0" />
+            <CheckBox Name="Us" Content="&#xA;" Grid.Column="1" Margin="31,10,32,10" Grid.Row="1" HorizontalContentAlignment="Center" VerticalContentAlignment="Center" IsChecked="True" HorizontalAlignment="Center" VerticalAlignment="Center"/>
+            <Label Content="RU" Margin="6,6,3,6" Grid.Row="2" Grid.Column="0" />
+            <CheckBox Name="Ru" Content="&#xD;&#xA;" Grid.Column="1" Margin="31,10,32,10" Grid.Row="2" HorizontalContentAlignment="Center" VerticalContentAlignment="Center" IsChecked="True" HorizontalAlignment="Center" VerticalAlignment="Center"/>
+            <Label Content="TranslationDataRow" Margin="6" Grid.Row="3" Grid.ColumnSpan="2" Grid.Column="0"
+                   HorizontalContentAlignment="Center"/>
+            <Label Content="Source language" Grid.Row="4" Grid.Column="0" Margin="6,6,3,6"/>
+            <Label Content="Target language" Grid.Row="5" Grid.Column="0" Margin="6,6,3,6"/>
+            <Button Name="TranslateButton" Content="{Binding TranslateButtonContent}" IsEnabled="{Binding TranslateButtonIsEnabled}" Margin="6" Grid.Row="6" Grid.ColumnSpan="2" Grid.Column="0"
+                    Command="{Binding TranslatingButtonCommand}"/>
+            <ComboBox Name="SourceLanguage" Grid.Column="1" Margin="3,6,6,6" Grid.Row="4" SelectedIndex="0" HorizontalContentAlignment="Center" VerticalContentAlignment="Center" Text="{Binding SourceLanguageText, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}">
```

Comment:
I will use Enum.GetValues function to set ItemsSource property with enum in comboboxes.